--- libpreps-1.6.5/configure.in.orig	Sun Jan 27 01:50:24 2002
+++ libpreps-1.6.5/configure.in	Sun Jan 27 01:51:22 2002
@@ -85,7 +85,7 @@
 ], ac_plpgsql_dir=/usr/local/pgsql
 )
 
-plpgsql_dirs="$ac_plpgsql_dir/lib /usr/lib/pgsql /usr/lib /usr/local/lib /opt/postgres/lib"
+plpgsql_dirs="$ac_plpgsql_dir/lib /usr/lib/pgsql /usr/lib/pgsql/modules /usr/lib /usr/local/lib /opt/postgres/lib"
 AC_FIND_FILE(plpgsql.so, $plpgsql_dirs, plpgsql_dir)
 if test ! -r $plpgsql_dir/plpgsql.so; then
   AC_MSG_ERROR(plpgsql.so not found.  May need to use --with-plpgsqldir option)
