--- libpreps-1.9.0/configure.in.orig	2003-03-10 02:35:54.000000000 +0100
+++ libpreps-1.9.0/configure.in	2004-04-06 22:56:58.370806840 +0200
@@ -130,14 +130,8 @@
 #
 # PostgreSQL Libraries.
 #
-AC_MSG_CHECKING([PostgreSQL Libraries Installed])
-pgsql_libdirs="$ac_pgsql_dir/lib /usr/lib/pgsql /usr/lib /usr/local/lib /usr/lib/postgresql/lib /opt/postgres/lib"
-AC_FIND_FILE(libpq.a, $pgsql_libdirs, pgsql_libdir)
-if test ! -r $pgsql_libdir/libpq.a; then
-  AC_MSG_RESULT(no) 
-  AC_MSG_ERROR(libpq.a not found.  May need to use --with-pgsqldir option)
-fi                                                                              
-PG_LIBS="-L$pgsql_libdir -lpq -lcrypt"
+
+AC_CHECK_LIB(pq, main, [PG_LIBS="-lpq"], [AC_MSG_ERROR(libpq not found)])
 AC_MSG_RESULT(yes) 
 
 
