--- libpreps-1.6.5.patched/configure.in	Sun Jan 27 19:59:56 2002
+++ libpreps-1.6.5/configure.in	Sun Jan 27 20:30:31 2002
@@ -164,14 +164,6 @@
 test "x$prefix" = xNONE && prefix=$ac_default_prefix
 preps_datadir=$(echo $datadir | sed "s%\${prefix}%$prefix%")
 AC_DEFINE_UNQUOTED(DATA_DIR, "$preps_datadir/libpreps")
-
-AC_OUTPUT([Makefile 
-           preps/Makefile 
-           scripts/Makefile 
-           doc/Makefile
-           doc/html/Makefile
-           doc/C/Makefile])
-
 dnl
 dnl We need to remove the ${prefix} and ${exec_prefix} junk that will work
 dnl in the Makefiles, but will not work in our scripts...
@@ -200,3 +192,11 @@
 AC_OUTPUT_COMMANDS([chmod a+x scripts/create_prepsdb])
 AC_OUTPUT_COMMANDS([chmod a+x scripts/update_prepsdb])
 AC_OUTPUT_COMMANDS([chmod a+x scripts/create_schema])
+
+AC_OUTPUT([Makefile 
+           preps/Makefile 
+           scripts/Makefile 
+           doc/Makefile
+           doc/html/Makefile
+           doc/C/Makefile])
+
