--- libpreps-1.9.0/configure.in.orig	2004-04-06 23:07:44.000000000 +0200
+++ libpreps-1.9.0/configure.in	2004-04-06 23:13:52.449643424 +0200
@@ -150,7 +150,7 @@
          [Define if the PostgreSQL server headers are in their own directory.])
    fi
    AC_MSG_RESULT(yes)
-   PG_CFLAGS="-I$pgserv_incdir $PG_CFLAGS"
+   PG_CFLAGS="-I$pgserv_incdir -I$pgserv_incdir/server $PG_CFLAGS"
 else
    AC_DEFINE(PG_SERVER_DIRECTORY, 1,
       [Define if the PostgreSQL server headers are in their own directory.])
